smarts/sstudio/genscenario.py

```diff
@@ -243,11 +246,12 @@ def gen_scenario(
     ):
         with timeit("traffic", logger.info):
             for name, traffic in scenario.traffic.items():
+                derived_seed = random.randint(-0b111111111111111, 0b111111111111111)
```

Comment:
1. Does the use of `random.randint()` here have any determinism issues between diferent runs?
2. If different seeds are required, perhaps we can fix them as `derived_seed = global_seed + loop_iteration_number`.
3. Instead of binary inputs to `random.randint()`, we can simply use integer inputs.

Reply:
1. `random.randint()` does not cause determinism issues because it is pseudo-random.
2. That could also work.
3. If I take suggestion 2 this point would be already handled.